realm/realm-library/src/main/java/io/realm/internal/rxjava/AndroidSchedulers.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Just to be on the safe side, I think you should mention who the copyright holder is.

Reply:
These files are removed again. Nabil had a better idea for including that I completely forgot was possible.